lib/lib_sms.php

```diff
@@ -187,14 +186,20 @@ function sms_getActiveCalls($phone, &$calls, &$error)
 	// retrieve the calls
 	$error = '';
 	$calls = array();
-
-	_sms_getCallInfo(array("status" => "queued", "to" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "ringing", "to" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "in-progress", "to" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "queued", "from" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "ringing", "from" => $phone), $client, $calls, $error);
-	_sms_getCallInfo(array("status" => "in-progress", "from" => $phone), $client, $calls, $error);
-
+	
+	$statuses = array("queued", "ringing", "in-progress");
```

Comment:
Is it possible to get the array values from the twilio api? would be DRYer, not a blocker for merging